hadoop-ozone/s3gateway/pom.xml

```diff
@@ -166,6 +166,10 @@
       <groupId>org.apache.ozone</groupId>
       <artifactId>ozone-common</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>ozone-filesystem-common</artifactId>
+    </dependency>
```

Comment:
Sorry, I didn't realize `OzoneClientUtils` was in `ozonefs-common`.  Instead of adding the dependency, can you please move it and its test to `ozone-client`?